wallet/types/src/wallet_tx.rs

```diff
@@ -15,45 +15,37 @@
 
 use serialization::{Decode, Encode};
 
-use common::chain::{GenBlock, Transaction};
+use common::chain::{Block, Transaction};
 use common::primitives::id::WithId;
-use common::primitives::Id;
-
-#[derive(Debug, PartialEq, Eq, Clone, Decode, Encode)]
-pub enum TxState {
```

Comment:
Why is this being removed? This is really important information for the user... to know whether their transaction is confirmed or has a problem.

Reply:
Reverted, thanks.